pipeline/clean/opuscleaner/clean-corpus.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
```

Comment:
Thought: This command is testable as well, but I'm not going to request a test here at this time.